### PR TITLE
E2e update: use new avd names, update e2e/README.md

### PR DESCRIPTION
**Pull request by fluidvanadium**

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-compare to the configuration aliases in `.detoxrs`
 pick a test called `e2e/TESTNAME.test.js`
-`yarn detox build TESTNAME -c CONFIGURATION`
-`yarn detox test TESTNAME -c CONFIGURATION`
+`yarn detox build -c android.emu.debug`
+`yarn detox test TESTNAME -c android.emu.debug`
+
+alt)
+`./scripts/start_flow_emulator.sh -a armeabi-v7a`
+`yarn detox test -c android.emu.32`
```
Comment (Oscar-Pepper) [inline e2e/README.md]: test should be after build right?

also i think this should be the default and you should explain how to run a specific test afterwards, what do you think?

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]

File: e2e/README.md (hunk 1 above)
Comment (fluidvanadium) [reply e2e/README.md]: oops, i guess i wasnt clear

Comment (Oscar-Pepper) [reply e2e/README.md]: test is still before build...

also what do you think about having the instructions explain how to run all tests and then have a:
`run a specific test)`
pointer at the end ?

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]

File: e2e/README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-`$ emulator -list-avds`
-compare to the configuration aliases in `.detoxrs`
-pick a test called `e2e/TESTNAME.test.js`
-`yarn detox build TESTNAME -c CONFIGURATION`
-`yarn detox test TESTNAME -c CONFIGURATION`
+to run a specific test)
+`yarn detox test TESTNAME -c android.emu.debug`
+
+alternative flow for testing a 32-bit architecture. make sure the -c configuration matches the emulator
+`./scripts/start_flow_emulator.sh -a armeabi-v7a`
```
Comment (Oscar-Pepper) [inline e2e/README.md]: change this to x86

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]

**Review by Oscar-Pepper — APPROVED** [review]

**Pull request merged**